### ISSUE TITLE
 internationalization

### ISSUE THREAD

**Issue by 214511455** [openstf/stf#315]
i want to add some langs supports 

1 . I modify res/commom/lang/langs.json , and add my lang key 
2. I add   "stf.th.po "  in   "res\common\lang\po"
3. I add "stf.th.po"  in  "res\common\lang\translations"
4. I reboot the stf
but it doesn't works , is there somting i need to do????

thank you for your help

**Comment by gunta** [comment]
If you want to test you need to run `gulp translate`.

Note that we are managing all the translations at Transifex since its better than just plain Github for contributing.

I've created the Thai language over it, so can you upload the translated file to [https://www.transifex.com/openstf/stf/]() ? You will need to register and apply for the Thai language only once, then you and other contributors will be able to do all the translations online.

After that you'll be able to send a pull request following the steps [here](https://github.com/openstf/stf#translating).

**Comment by 214511455** [comment]
i modify the langs.json  and other lang files (or some js file)  , does it no auto reload that files?

**Comment by gunta** [comment]
No, you need to run `gulp translate` after that if you want to check.

But you can skip this, just upload the `po` to Transifex and we can do the rest.
